packages/fxa-payments-server/Gruntfile.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = function (grunt) {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    concat: {
+      ftl: {
+        src: [
+          'src/lib/l10n/branding.ftl',
+          'src/*.ftl',
+          'src/lib/*.ftl',
+          'src/routes/**/*.ftl',
+          'src/components/**/*.ftl',
+        ],
+        dest: 'public/locales/en/payments.ftl',
+      },
+
+      // We need this for tests because we pull the latest from `fxa-content-server-l10n`
```

Comment:
l10n testing has not yet been added for payments-server

Reply:
Payments-server is not yet using the l10n test utilities we've begun to define in `fxa-react`, but it does have some l10n testing with [`setupFluentLocalizationTest`](https://github.com/mozilla/fxa/blob/81d18892e22c1e0e1efec96b1270251bbf3e92f2/packages/fxa-payments-server/src/lib/test-utils.tsx#L695) and `getLocalizedMessage`.

If we leave `setupFluentLocalizationTest` as-is, it's going to attempt to pull from the `public/locales` dir, and it'll test against the l10n side's latest and not our latest. We need to set up that `merge-ftl:test` script in `package.json` and have it run with the test command, and then we should remove `setupFluentLocalizationTest` in favor of using `fxa-react`'s `getFtlFromPackage`. There's [a comment](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/lib/test-utils/index.ts#L45) in that file about doing this in FXA-5996. You'll need to update any function needing this to be `async`.

I think we can leave `getLocalizedMessage` for now, and have other issues filed for the new testing methods after this and FXA-6001 are merged.

Reply:
@vpomerleau Your changes look great, we still need to add the file to `.gitignore` though; we don't want to commit `test/payments.ftl`. We should also be able to remove `setupFluentLocalizationTest`.

Reply:
Right, I'll clean that `test` folder from the commit. 
I'd removed `setupFluentLocalizationTest` already in a small extra push, probably just as you were reviewing :)

Reply:
@vpomerleau I think we still need to add the test directory to the `.gitignore`.

Reply:
There was also a .gitignore template in the /src folder - I've removed that one.

Reply:
@LZoog Most are covered in the global `gitignore`. I'll admit I remove this a bit too fast - here are the lines that weren't in the global:
```
.env.local
.env.development.local
.env.test.local
.env.production.local
/.pnp
.pnp.js
```
What would you think if I move these to global and still remove the file in `/src`?

Reply:
I've addressed the edits from [FXA-6255](https://mozilla-hub.atlassian.net/browse/FXA-6255) in my latest commit.